libobs/obs-audio-controls.h

```diff
@@ -259,6 +259,8 @@ EXPORT void obs_volmeter_add_callback(obs_volmeter_t *volmeter,
 EXPORT void obs_volmeter_remove_callback(obs_volmeter_t *volmeter,
 		obs_volmeter_updated_t callback, void *param);
 
+EXPORT float log_def_to_db(const float def);
```

Comment:
This function is supposed to be internal and not necessarily the one used be the fader/volmeter to convert between dB and deflection. The function that is used is depends on `obs_fader_type`.